python/mxnet/ndarray/sparse.py

```diff
@@ -1176,6 +1480,7 @@ def zeros(stype, shape, ctx=None, dtype=None, **kwargs):
     >>> mx.nd.sparse.zeros('row_sparse', (1,2), ctx=mx.cpu(), dtype='float16').asnumpy()
     array([[ 0.,  0.]], dtype=float16)
     """
+    # pylint: disable= no-member, protected-access
```

Comment:
Is this added to every function? Maybe worth disable it at the beginning of the file instead of disabling it per function.

Reply:
Not added to every function, so will keep this in this way.